modules/bliinkBidAdapter.js

```diff
@@ -75,79 +84,51 @@ export function getKeywords() {
     ]
 
     if (keywords && keywords.length > 0) {
-      return keywords
-        .filter((value) => value)
-        .map((value) => value.trim())
+      return keywords.filter((value) => value).map((value) => value.trim());
     }
   }
 
-  return []
-}
-
-export const parseXML = (content) => {
-  if (typeof content !== 'string' || content.length === 0) return null
-
-  const parser = new DOMParser()
-  let xml;
-
-  try {
-    xml = parser.parseFromString(content, 'text/xml')
-  } catch (e) {}
-
-  if (xml &&
-    xml.getElementsByTagName('VAST')[0] &&
-    xml.getElementsByTagName('VAST')[0].tagName === 'VAST') {
-    return xml
-  }
-
-  return null
+  return [];
 }
 
 /**
  * @param bidRequest
- * @param bliinkCreative
- * @return {{cpm, netRevenue: boolean, requestId, width: (*|number), currency, ttl: number, creativeId, height: (*|number)} & {mediaType: string, vastXml}}
+ * @return {({cpm, netRevenue: boolean, requestId, width: number, currency, ttl: number, creativeId, height: number}&{mediaType: string, vastXml})|null}
  */
-export const buildBid = (bidRequest, bliinkCreative) => {
-  if (!bidRequest && !bliinkCreative) return null
-
-  const body = {
-    requestId: bidRequest.bidId,
-    currency: bliinkCreative.currency,
-    cpm: bliinkCreative.price,
-    creativeId: bliinkCreative.creativeId,
-    width: (bidRequest.sizes && bidRequest.sizes[0][0]) || 1,
-    height: (bidRequest.sizes && bidRequest.sizes[0][1]) || 1,
-    netRevenue: false,
-    ttl: 3600,
-  }
-
-  // eslint-disable-next-line no-mixed-operators
-  if ((bliinkCreative) && bidRequest &&
-    // eslint-disable-next-line no-mixed-operators
-    !bidRequest.bidId ||
-    !bidRequest.sizes ||
-    !bidRequest.params ||
-    !(bidRequest.params.placement)
-  ) return null
-
-  delete bidRequest['bids']
+export const buildBid = (bidRequest) => {
+  if (!bidRequest || !bidRequest.creative || !bidRequest.creative.media_type) return null;
 
-  switch (bliinkCreative.media_type) {
+  let bid;
+  switch (bidRequest.creative.media_type) {
     case VIDEO:
-      return Object.assign(body, {
-        mediaType: VIDEO,
-        vastXml: bliinkCreative.content,
-      })
+      const vastXml = bidRequest.creative.video.content
```

Comment:
This line may benefit from using `deepAccess` to access `creative.video.content` from `bidRequest`